app/cluster-api-provider-sidero/main.go

```diff
@@ -26,6 +28,10 @@ import (
 	// +kubebuilder:scaffold:imports
 )
 
+const (
+	debugAddr = ":9994"
```

Comment:
Port numbers are sequential but basically in the random range.

Thanks to `debug.Enabled` being a constant, port number can be configurable via flags, environment variables, etc. Should it be?

Reply:
I think hardcoded should be fine?